### PR TITLE
Dp 23849 add collection search to all content and my content view

### PR DESCRIPTION
**Pull request by beto-aveiga**
**Description:**
- Adds autocomplete collections filter to view content and my_content.

**Jira:** (Skip unless you are MA staff)
DP-23849

---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)

### CODE REVIEW

File: conf/drupal/config/views.view.content.yml [hunk 1]
```diff
@@ -4118,17 +4180,15 @@ display:
         groups:
           1: AND
       exposed_form:
-        type: input_required_on_request
+        type: basic
```
Comment (arthurbaghdas) [inline conf/drupal/config/views.view.content.yml]: @beto-aveiga I think you reverted your prev changes.

Comment (beto-aveiga) [reply conf/drupal/config/views.view.content.yml]: Thanks! Good catch, you are probably right. I will fix that quickly.

Comment (beto-aveiga) [reply conf/drupal/config/views.view.content.yml]: Fixed. I will wait until this test passes: Drupal.Tests.mass_views.ExistingSiteJavascript.AllContentViewTest

File: conf/drupal/config/views.view.content.yml [hunk 2]
```diff
@@ -4121,14 +4183,12 @@ display:
...
-          reset_button: 1
+          reset_button: true
           reset_button_label: Reset
           exposed_sorts_label: 'Sort by'
-          expose_sort_order: 1
+          expose_sort_order: true
           sort_asc_label: Asc
           sort_desc_label: Desc
-          text_input_required: 'Select any filter and click on Apply to see results'
-          text_input_required_format: basic_html
```
Comment (arthurbaghdas) [inline conf/drupal/config/views.view.content.yml]: This part doesnt seem relevant as well @beto-aveiga

Comment (beto-aveiga) [reply conf/drupal/config/views.view.content.yml]: You are right @arthurbaghdas. Thanks again!

Comment (beto-aveiga) [reply conf/drupal/config/views.view.content.yml]: Fixed. 
I trusted on the test passing too much.
I checked the changed files this time. Everything looks intended.

**Pull request merged**